optimize-wasm/optimize-wasm.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+if [ -z "$1" ]
+  then
+    echo "Usage: $0 <filename.wasm>"
+    exit
+fi
+
+new=`basename $1 .wasm`-optimized.wasm
```

Comment:
```suggestion
new="$(basename "$1" .wasm)"-optimized.wasm
```

Backticks are considered legacy and inputs should be properly quoted.

As a tip, you might want to take a look at Shellcheck. It is quite helpful when writing shell scripts.